### PR TITLE
adding Strong 1911

### PR DESCRIPTION
**Pull request by SimonGreenhill**
# Pull request checklist

- [X] add new concept list

## Additional information

Note that the author's name is "W. Mersh Strong", not "Marsh" as the typo in both the journal and the other metadata out online suggest.

### CODE REVIEW

**Comment by chrzyki** [comment]
Thank you for adding this list! @mathildavz, do you have time for a review? I can also help with reviewing.

**Comment by mathildavz** [comment]
@chrzyki I am currently on sick leave, but I will gladly help with the review as soon as I'm back to work, if it's not too time sensitive...

**Comment by chrzyki** [comment]
> @chrzyki I am currently on sick leave, but I will gladly help with the review as soon as I'm back to work, if it's not too time sensitive...

No worries, hope you get better soon!

@SimonGreenhill Would you be okay with this PR being in the pipeline for a bit longer? I'd start a review soon and then we wait until @mathildavz is back. Or should I ask someone else to get this done quickly?

**Comment by SimonGreenhill** [comment]
yes, no rush on my part.

File: concepticondata/conceptlists.tsv [hunk 1]
```diff
@@ -431,4 +431,5 @@ Auderset-2023-240	Auderset, Sandra,  and Greenhill, Simon J., and DiCanio, Chris
...
\ No newline at end of file
+Tao-2023-90	Tao, Yuxin and Wei, Yuancheng and Ge, Jiaqi and Pan, Yan and Wang, Wenmin and Bi, Qianqi and Sheng, Pengfei and Fu, Changzhong and Pan, Wuyun and Jin, Li and Zheng, Hong-Xiang and Zhang, Menghan	2023		90	basic	English	Kra-Dai Languages	https://static-content.springer.com/esm/art%3A10.1038%2Fs41467-023-42761-x/MediaObjects/41467_2023_42761_MOESM1_ESM.pdf	Tao2023		A 90 item wordlist based on [Swadesh 100](:ref:Swadesh-1955-100), the items removed were those that were highly borrowed or were not distinguished semantically in the Kra-Dai (Tai-Kadai) languages.		
+Strong-1911-150	W. Mersh Strong	1911		150	basic	English	New Guinea languages		Strong1911		A brief vocabulary of 150 basic items for the Kaki Ae (Tait) language.	178-181
```
Comment (chrzyki) [inline concepticondata/conceptlists.tsv:435]: Order of author column: `Strong, Walter Mersh`

File: concepticondata/conceptlists/Strong-1911-150.tsv [hunk 2]
```diff
@@ -0,0 +1,151 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Strong-1911-150-1	1	Adze	290	ADZE
+Strong-1911-150-2	2	Areca nut
```
Comment (chrzyki) [inline concepticondata/conceptlists/Strong-1911-150.tsv:3]: We do have a number of lists that map this to `217 BETELNUT`.

File: concepticondata/conceptlists/Strong-1911-150.tsv [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+Strong-1911-150-67	67	Hear	1408	HEAR
+Strong-1911-150-68	68	Hill	614	HILL
+Strong-1911-150-69	69	Hook	1071	HOOK
+Strong-1911-150-70	70	Hot	1286	HOT
+Strong-1911-150-71	71	House	1252	HOUSE
+Strong-1911-150-72	72	Husband	1200	HUSBAND
+Strong-1911-150-73	73	Kangaroo	164	KANGAROO
+Strong-1911-150-74	74	Know	3626	KNOW
+Strong-1911-150-75	75	Leaf	628	LEAF
+Strong-1911-150-76	76	Log	1803	WOOD
```
Comment (chrzyki) [inline concepticondata/conceptlists/Strong-1911-150.tsv:77]: There's quite a number of lists that have *log* and most of them are unmapped. I think here I'd rather opt for introducing a new concept (and mapping the other lists) than mapping to `WOOD`.

File: concepticondata/conceptlists/Strong-1911-150.tsv [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+Strong-1911-150-77	77	Lime	2540	LIME (MATERIAL)
+Strong-1911-150-78	78	Lip	478	LIP
+Strong-1911-150-79	79	Live	1422	BE ALIVE
```
Comment (chrzyki) [inline concepticondata/conceptlists/Strong-1911-150.tsv:80]: Can this be disambiguated version `363 DWELL (LIVE, RESIDE)`? From a first glance, `363` might be a bitter fit.

File: concepticondata/conceptlists/Strong-1911-150.tsv [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+Strong-1911-150-89	89	Name	1405	NAME
+Strong-1911-150-90	90	Navel	1838	NAVEL
+Strong-1911-150-91	91	Near	1942	NEAR
+Strong-1911-150-92	92	Night	1233	NIGHT
+Strong-1911-150-93	93	Nipple	796	NIPPLE
+Strong-1911-150-94	94	No	1269	NO
+Strong-1911-150-95	95	Nose	1221	NOSE
+Strong-1911-150-96	96	Paddle	234	PADDLE
+Strong-1911-150-97	97	Pig	1337	PIG
+Strong-1911-150-98	98	Rain	2108	RAINING OR RAIN
```
Comment (chrzyki) [inline concepticondata/conceptlists/Strong-1911-150.tsv:99]: Maybe rather `658`?

File: concepticondata/conceptlists/Strong-1911-150.tsv [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+Strong-1911-150-141	141	White	1335	WHITE
+Strong-1911-150-142	142	Wife	1199	WIFE
+Strong-1911-150-143	143	Wind	960	WIND
+Strong-1911-150-144	144	Woman	962	WOMAN
+Strong-1911-150-145	145	Yam	410	YAM
+Strong-1911-150-146	146	One	1493	ONE
+Strong-1911-150-147	147	Two	1498	TWO
+Strong-1911-150-148	148	Three	492	THREE
+Strong-1911-150-149	149	Four	1500	FOUR
+Strong-1911-150-150	150	Five	493	FIVE
```
Comment (chrzyki) [inline concepticondata/conceptlists/Strong-1911-150.tsv:151]: Towards the end of the list, I think there are the following additional concepts:

- *Yes*
- *Yellow*

**Review by chrzyki — CHANGES_REQUESTED** [review]

File: concepticondata/conceptlists/Strong-1911-150.tsv [hunk 7]
```diff
@@ -0,0 +1,151 @@
...
+Strong-1911-150-17	17	Bow (noun)	994	BOW
+Strong-1911-150-18	18	Branch	1531	BRANCH
+Strong-1911-150-19	19	Bring	692	BRING
+Strong-1911-150-20	20	Bury	1719	BURY
+Strong-1911-150-21	21	Butterfly	1791	BUTTERFLY
+Strong-1911-150-22	22	Centipede	1974	CENTIPEDE
+Strong-1911-150-23	23	Chest	1592	CHEST
+Strong-1911-150-24	24	Charcoal	1188	CHARCOAL
+Strong-1911-150-25	25	Child	1304	CHILD (YOUNG HUMAN)
+Strong-1911-150-26	26	Claw (of bird)	72	CLAW
```
Comment (mathildavz) [inline concepticondata/conceptlists/Strong-1911-150.tsv:27]: I'd unmap, a claw of a bird is more specific than our general `72 CLAW`.

**Review by mathildavz — COMMENTED** [review]
I've added two small comments with my review. Other than that the list looks great! Thank you for adding it @SimonGreenhill!

File: concepticondata/conceptlists/Strong-1911-150.tsv (hunk 3 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Strong-1911-150.tsv:77]: ok, i'll unmap.

File: concepticondata/conceptlists/Strong-1911-150.tsv (hunk 4 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Strong-1911-150.tsv:80]: I can't tell from the data which one it is. I think live is probably more likely? shall I unmap it?

File: concepticondata/conceptlists/Strong-1911-150.tsv (hunk 6 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Strong-1911-150.tsv:151]: oh, not sure how I missed them. Sorry, fixed.

**Comment by SimonGreenhill** [comment]
All changes requested made except for the unresolved comment re LIVE. Thoughts @mathildavz and @chrzyki ?

File: concepticondata/conceptlists/Strong-1911-150.tsv (hunk 4 above)
Comment (chrzyki) [reply concepticondata/conceptlists/Strong-1911-150.tsv]: Judging from the other mappings in comparable lists, if we can't say which of the two it is, I'd opt for unmapping.

**Comment by chrzyki** [comment]
Looks good to me now! Thanks for incorporating the changes.

**Review by chrzyki — APPROVED** [review]

File: concepticondata/conceptlists/Strong-1911-150.tsv (hunk 4 above)
Comment (SimonGreenhill) [reply concepticondata/conceptlists/Strong-1911-150.tsv]: ok

**Pull request merged**